### PR TITLE
Improves docs around Kubernetes Inventory UI

### PR DESCRIPTION
**Pull request by hev**
This PR updates the docs related to Kubernetes UI and makes Kubernetes Inventory more prominent. Includes the following changes.

* Highlights KAI on homepage
* Adds Using Kubernetes Inventory section
* Cleans up the concepts page to focus on installing kai and moves embedded mode to sub-page.

Would still like to add screenshots to the using page.

### CODE REVIEW

**Comment by dakaneye** [comment]
This probably needs to merge into 3.1-dev

File: content/_index.html [hunk 1]
```diff
@@ -19,8 +19,8 @@
 
 {{% blocks/section title="Integrations" subtitle="Integrate Anchore with your favorite CI/CD Systems, Orchestration Platforms, and more" color="primary" %}}
 
-{{% blocks/feature svg="icons/kubernetes-logo.svg" title="Kubernetes" %}}
-Install Anchore using the [Helm Chart](https://github.com/anchore/anchore-charts/blob/master/stable/anchore-engine) and make admission decisions based on analysis and policy-based evaluations with our [Kubernetes Admission Controller](https://github.com/anchore/kubernetes-admission-controller)
```
Comment (dakaneye) [inline content/_index.html:23]: not sure this should remove the reference to the actual Anchore Helm chart

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 2]
```diff
@@ -1,169 +1,62 @@
...
+  # Path should not be changed, cluster value is used to tell Anchore which cluster this inventory is coming from
+  kubeconfig:
+    cluster: <unique-name-for-your-cluster>
 
-If you would prefer to not provide Kubernetes API access to Anchore directly, `kai` can be configured to run as an agent within your cluster via a helm chart, and can also run on it's own as a CLI tool (for more information about the CLI itself, head over to the [kai repo](https://github.com/anchore/kai).
+  anchore:
+    url: <URL for your>
 
-`kai`'s helm chart is hosted as part of the https://charts.anchore.io repo. It is based on the [anchore/kai](https://hub.docker.com/r/anchore/kai) docker image. 
+    # Note: reccommend using the inventory-agent role
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md:30]: nit: typo

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 3]
```diff
@@ -1,169 +1,62 @@
...
+    user: <user>
+    password: <password>
+```
 
-To install the helm chart, follow these steps:
+2. Run helm install in the cluster(s) you wish to track
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md:35]: nit: install "for" the cluster(s) you wish to track

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 4]
```diff
@@ -1,169 +1,62 @@
...
-To install the helm chart, follow these steps:
+2. Run helm install in the cluster(s) you wish to track 
 ```
 $ helm repo add anchore https://charts.anchore.io
 $ helm install <release> -f <values.yaml> anchore/kai
 ``` 
-An example values file can be found [here](https://github.com/anchore/anchore-charts/tree/master/stable/kai/values.yaml).
 
-The key configurations are in the `kai.anchore` section. Kai must be able to resolve the Anchore URL and requires API credentials.
+Kai must be able to resolve the Anchore URL and requires API credentials. Review the kai logs if you are not able to see the inventory results in the UI.
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md:41]: external API and catalog service logs may also be consulted for troubleshooting purposes.

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 5]
```diff
@@ -1,169 +1,62 @@
...
 
 Note: the Anchore API Password can be provided via a kubernetes secret, or injected into the environment of the `kai` container
 * For injecting the environment variable, see: `inject_secrets_via_env`
 * For providing your own secret for the Anchore API Password, see: `kai.existing_secret`. `kai` creates it's own secret based on your values.yaml file for key `kai.anchore.password`, but the `kai.existingSecret` key allows you to create your own secret and provide it in the values file.
 
-# Image Time-To-Live
-As part of reporting images in your runtime environment, Anchore maintains an active record of that set of images based on when the Image has been last reported by `kai`.
+## Agentless mode
+
+In addition to running as agent kai can also be run by Anchore Enterpise in an embedded mode. In this mode `kai` runs within the Anchore Enterprise, and needs a cluster configuration to access the Kubernetes API. This still requires cluster but allows for monitoring without the installation of additional components. See more details about setting up [embedded mode](./embedded/).
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md:49]: within the Anchore Enterprise "container"

Comment (dakaneye) [reply content/docs/overview/concepts/images/inventory/_index.md:49]: requires "a" cluster

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 6]
```diff
@@ -1,169 +1,62 @@
...
 
-# Image Time-To-Live
-As part of reporting images in your runtime environment, Anchore maintains an active record of that set of images based on when the Image has been last reported by `kai`.
+## Agentless mode
+
+In addition to running as agent kai can also be run by Anchore Enterpise in an embedded mode. In this mode `kai` runs within the Anchore Enterprise, and needs a cluster configuration to access the Kubernetes API. This still requires cluster but allows for monitoring without the installation of additional components. See more details about setting up [embedded mode](./embedded/). 
+
+## CLI and API Inventory Collection
+In situations where you may not be able to have consitent network connection `kai` along with the inventory API can be used to collect and report inventory back to Anchore Enterprise. Inventory collection can also be used to collect other types of containers runtimes however non are officially supported at this time.
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md:52]: last sentence is a bit wordy: "We hope to support other types of container runtime inventory in the future"

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 7]
```diff
@@ -1,169 +1,62 @@
...
-```
-
-### Usage
-Once a cluster is configured, Anchore will execute the kai binary on an interval, parse the JSON output, and then store the current snapshot of the inventory.
-That inventory may be listed by running HTTP GET on `/v1/enterprise/inventories`.
-
-To configure the interval on which the kai binary runs, adjust the value of the `ANCHORE_CATALOG_K8S_INVENTORY_REPORT_INTERVAL_SEC` environment variable on the enterprise catalog service. By default, it runs every 5 minutes.
+To install the helm chart, follow these steps:
 
-Clusters may be retrieved in list format via HTTP GET on `/v1/enterprise/inventories/clusters` or by name, with HTTP GET on `v1/enterprise/inventories/clusters/<name>`
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md:133]: this bit seems to have not made it back into the embedded mode documentation

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 8]
```diff
@@ -1,169 +1,62 @@
...
+## CLI and API Inventory Collection
+In situations where you may not be able to have consitent network connection `kai` along with the inventory API can be used to collect and report inventory back to Anchore Enterprise. Inventory collection can also be used to collect other types of containers runtimes however non are officially supported at this time. 
+
+### Usage
+To quickly verify that you are tracking Kubernetes Inventory you can quickly access inventory results from the `anchorectl` with the  command `anchorectl inventory list`. The UI for the Kubernetes allows operators to visually navigate vulnerability resultes, source where vulnerabilties come from and apply their default policy to thier runtime. You'll be able to subscribe to changes for your clusters, quicky asses vulnerabilities and more.
+
+For more details about watching clusters, and reviewing policy results see the [Using Kubernetes Inventory](/docs/using/ui_usage/kubernetes/) section.
+
 
-If `kai` does not find an image that was previously in the inventory, it waits for a configurable TTL to expire before removing that image:
```
Comment (dakaneye) [inline content/docs/overview/concepts/images/inventory/_index.md]: this section appears to be missing in the new documentation version as well

**Review by dakaneye — CHANGES_REQUESTED** [review]

**Comment by hev** [comment]
Wrong branch. Closing.

**Pull request closed without merging**